cmds/bmp/tasks_command.go

```diff
@@ -51,5 +55,41 @@ func (cmd tasksCommand) Validate() (bool, error) {
 
 func (cmd tasksCommand) Execute(args []string) (int, error) {
 	cmd.printer.Printf("Executing %s comamnd: args: %#v, options: %#v", cmd.Name(), args, cmd.options)
+
+	var latest uint
+	latest = 50
+	if cmd.options.Latest != 0 {
+		latest = cmd.options.Latest
+	}
+
+	tasksResponse, err := cmd.bmpClient.Tasks(latest)
+	if err != nil {
+		return 1, err
+	}
+
+	if tasksResponse.Status != 200 {
+		return tasksResponse.Status, nil
+	}
+
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"Task ID", "Status", "Description", "Start", "End"})
+
+	length := len(tasksResponse.Data)
+	content := make([][]string, length)
+	for i, task := range tasksResponse.Data {
+		content[i] = []string{
+			strconv.Itoa(task.Id),
+			task.Description,
+			task.Status,
+			task.Start_time,
```

Comment:
Why is this named `Start_time` vs `StartTime`? Motivate this choice, please. Same as `End_time`